spec/lib/metrics_logger_spec.rb

```diff
@@ -1,7 +1,7 @@
-require 'spec_helper'
+require 'rails_helper'
```

Comment:
Another pretty long spec.  As per my previous Sandi Metz comment, I'd like to see it broken out or refactored to make it more compact...